google-cloud-trace/lib/google/cloud/trace/utils.rb

```diff
@@ -32,10 +32,11 @@ def self.time_to_grpc time
 
           # This is called with different types of objects. Extract
           # the nanoseconds appropriately.
-          nanos = case time.class
-                  when Time
+          puts "SEPH: ", time.class, time
```

Comment:
Maybe you left this debug `puts` by mistake?

Reply:
oops. Thanks!